mailpoet/lib/Form/PreviewPage.php

```
@@ -8,7 +8,7 @@
 
 class PreviewPage {
   const PREVIEW_DATA_TRANSIENT_PREFIX = 'mailpoet_form_preview_';
-  const PREVIEW_DATA_EXPIRATION = 84600; // 1 DAY
+  const PREVIEW_DATA_EXPIRATION = DAY_IN_SECONDS;
```

PHPStan (2.2.8)

[error] Constant DAY_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)